Unable to get greenworks to build on abi > 87 on MacOs

The error: 
```
In file included from ../src/api/greenworks_api_utils.cc:7:
In file included from ../node_modules/nan/nan.h:56:
In file included from /Users/runner/Library/Caches/node-gyp/12.0.2/include/node/node.h:67:
In file included from /Users/runner/Library/Caches/node-gyp/12.0.2/include/node/v8.h:30:
/Users/runner/Library/Caches/node-gyp/12.0.2/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
```

It appear here: https://github.com/ElectronForConstruct/greenworks-prebuilds/runs/2232623990?check_suite_focus=true (Go to "Build")

I've found some issues about this already: https://github.com/microsoft/node-pty/issues/441

Maybe it's related to this: 
- https://github.com/greenheartgames/greenworks/blob/master/binding.gyp#L135
- https://github.com/greenheartgames/greenworks/blob/master/binding.gyp#L171

--- a/binding.gyp
+++ b/binding.gyp
@@ -132,7 +132,7 @@
           }],
         ],
       },
-      'cflags': [ '-std=c++11' ],
+      'cflags': [ '-std=c++14' ],
       'conditions': [
         ['OS== "linux"',
           {
@@ -168,7 +168,7 @@
           '-Wno-deprecated-declarations',
         ],
         'OTHER_CPLUSPLUSFLAGS' : [
-          '-std=c++11',
+          '-std=c++14',
           '-stdlib=libc++'
         ],
         'OTHER_LDFLAGS': [

